### PR TITLE
meson: remove libplacebo wrap

### PR DESCRIPTION
**Pull request by llyyr**

### CODE REVIEW

**Review by Dudemanguy — APPROVED** [review]
I like this solution.

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1 @@
-/subprojects/*
-!/subprojects/packagefiles
-!/subprojects/*.wrap
+/subprojects
```
Comment (kasper93) [inline .gitignore:1]: please don't revert that. Just keep it. Or remove also `/subprojects`

**Comment by kasper93** [comment]
It's fine, I though wrap would be also useful for users to build mpv easily without the need to manually provide libplacebo first, but since we don't like wraps. (and we don't do them for any other dep) so it make sense.

File: .gitignore (hunk 1 above)
Comment (Dudemanguy) [reply .gitignore]: We probably wouldn't use it but yeah I guess it's better to just keep it the way it was before.

File: meson.build [hunk 2]
```diff
@@ -23,8 +23,7 @@ libavutil = dependency('libavutil', version: '>= 56.70.100')
 libswresample = dependency('libswresample', version: '>= 3.9.100')
 libswscale = dependency('libswscale', version: '>= 5.9.100')
```
Comment (kasper93) [inline meson.build:25]: nitpick, but this empty line is not necessary

Comment (Dudemanguy) [reply meson.build:25]: Well it was to logically separate it from the ffmpeg dependencies I presume.

Comment (llyyr) [reply meson.build:25]: There was originally a space between ffmpeg deps and libass, so I kept that. I think it makes sense

Comment (kasper93) [reply meson.build:25]: I'm stupid, don't mind me. I though they are grouped together.

**Review by haasn — APPROVED** [review]

**Pull request merged**